### PR TITLE
CE-3374 Display A Wikia contributor instead of an IP of an anonymous user

### PR DESCRIPTION
**Pull request by adamkarminski**
## Links

* http://wikia-inc.atlassian.net/browse/CE-3374

## Description

Display _A Wikia contributor_ instead of an IP of an anonymous user. Also, actually **pass** an information that the user is anonymous.

## Reviewers

@Wikia/community-engineering

### CODE REVIEW

File: front/common/public/locales/en/main.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "app": {
+    "anonymous-user": "A Wikia contributor",
```
Comment (Grunny) [inline front/common/public/locales/en/main.json]: This is already available (and translated) in the message `username-anonymous`.

Comment (adamkarminski) [inline front/common/public/locales/en/main.json]: D'oh!

Comment (adamkarminski) [inline front/common/public/locales/en/main.json]: Updated :)

File: front/main/app/models/article-diff.js [hunk 2]
```diff
@@ -86,6 +88,7 @@ ArticleDiffModel.reopenClass({
 
 				if (diffs) {
 					modelInstance = ArticleDiffModel.create({
+
```
Comment (Grunny) [inline front/main/app/models/article-diff.js]: Random blank line? :)

**Comment by Grunny** [comment]
Super minor comment, but LGTM. :)

**Comment by adamkarminski** [comment]
@Grunny - I updated the PR :)

**Comment by lukaszkonieczny** [comment]
LGTM

**Comment by azarek** [comment]
LGTM

**Pull request merged**